docs/pages/concepts/source.md

```diff
@@ -26,6 +26,11 @@ table named `user` within the Postgres database. And that's it - once this
 is written, `UserLocation` dataset will start mirroring your postgres table
 `user`and will update as the underlying Postgres table updates.
 
+### Bounded
```

Comment:
I think we should also add an example of how bounded and idleness will most likely be used i.e. 

One batch source with bounded and idleness set. Also a comment saying something like 'after initial ingestion, this data source is considered/marked as closed due to no data flow for `idleness` time period'.
One stream source 

There is a small pipeline which say does filter first on each individually say on timestamp column so avoid duplicate data. Something like -

@pipeline(ds_batch, ds_stream)
def pipe():
   ds_batch = ds_batch.filter('ts' < '2024-02-20')
   ds_stream = ds_stream.filter('ts' >= '2024-02-20')
    return ds_batch + ds_stream

Reply:
Updated the documentation. Let me know if it looks good